src/nupic/movingaverage.capnp

```diff
@@ -4,5 +4,5 @@
 struct MovingAverageProto {
   windowSize @0 :UInt32;
   slidingWindow @1 :List(Float32);
-  total @2 :UInt32;
+  total @2 :Float32;
```

Comment:
Do we need to make this a different field? Or do we not care about backwards-compatibility at this point?